code/game/objects/machinery/robotic_cradle.dm

```diff
@@ -127,8 +123,8 @@
 			reason = "Reason for discharge: Destruction of linked CRADLE Engineering System. Alerting security advised."
 	radio.talk_into(src, "<b>Patient: [occupant] has been released from [src] at: [get_area(src)]. [reason]</b>", RADIO_CHANNEL_MEDICAL)
 	occupant = null
-	update_icon()
 	stop_processing()
+	update_icon()
```

Comment:
You don't need to do this since you fixed update_icon_state()

Reply:
tested, still need this otherwise there is nothing to call the icon change when a synth leaves the cradle